packages/runtime/src/helpers/verification.ts

```diff
@@ -105,7 +105,11 @@ export const checkForRootPublish = ({
   }
 }
 
-export const checkZipSize = async (file: string, maxSize: number = LAMBDA_MAX_SIZE): Promise<void> => {
+export const checkZipSize = async (
```

Comment:
Can we add a couple tests for this @biruwon? One for not calling console.log if `fileSize < warningSize` and one for checking that console.log is indeed called with the message or checking for a partial message if that's easier?

Currently we only have a test to check if the bundle size is disabled. Also, I can add those tests if you want.

See test/helpers/verification.spec.ts

Reply:
@biruwon I pushed test fixes - it was mix of things:
 - spying correct `console.X` method
 - clearing mocks so previous tests don't impact future tests
 - resetting mocks so next `describe` blocks that didn't want mocks are not borked
 - disabling chalk's coloring for testing as I couldn't get it to match any other way

Reply:
thanks @pieh for that! as I can't not approve my own PRs, can you approve it and move it forward? Like the thing you did for gastby